pkg/ingester/ingester.go

```diff
@@ -778,15 +778,6 @@ func (i *Ingester) Series(ctx context.Context, req *logproto.SeriesRequest) (*lo
 	return instance.Series(ctx, req)
 }
 
-// Check implements grpc_health_v1.HealthCheck.
-func (i *Ingester) Check(ctx context.Context, req *grpc_health_v1.HealthCheckRequest) (*grpc_health_v1.HealthCheckResponse, error) {
-	status := grpc_health_v1.HealthCheckResponse_SERVING
-	if (i.State() != services.Running) || (i.lifecycler.GetState() != ring.ACTIVE) {
```

Comment:
I'm a bit worried that we don't seem to be checking the lifecycler state in our blanket version, but it doesn't look like these are used either.